src/nvim/eval.c

```diff
@@ -12393,36 +12393,40 @@ static void f_jobstart(typval_T *argvars, typval_T *rettv, FunPtr fptr)
 }
 
 // "jobstop()" function
-static void f_jobstop(typval_T *argvars, typval_T *rettv, FunPtr fptr)
+// Returns 0, or -1 on error
```

Comment:
all of the `f_xx` functions are documented in `:help eval.txt` rather than in docstrings (although it would be nice if eval.txt were generated from these docstrings, but until someone implements that, keep it in one place ...)